styleguide.config.js

```diff
@@ -54,6 +54,10 @@ module.exports = {
       name: 'Modules',
       components: 'src/modules/!(select|modal)/**/*.{js,jsx,ts,tsx}',
       sections: [
+        {
+          name: 'Search result',
+          components: ['src/modules/search-result/search-result.jsx'],
```

Comment:
please exclude it here https://github.com/oacore/design/blob/master/styleguide.config.js#L55 So we don't have duplicate entries in menu